app/scripts/directives/util.js

```diff
@@ -50,6 +50,10 @@ angular.module('openshiftConsole')
           if (t && t.is('a')){
             return;
           }
+          // Also check if this is a child of an anchor element (e.g., an icon inside a link).
+          if (t && t.parent() && t.parent().is('a')) {
```

Comment:
this would be better http://api.jquery.com/closest/#closest-selector-context
```
t.closest("a", element)
```

it will look for the closest ancestor that is an `a` tag, but won't look farther up than the element (in this case the tile), then you can just see if the answer is empty

that will handle if you have a parent of a parent of a parent that is an a :)